src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,193 @@
+    } else if (description.length === 0) {
+      this.setState({ inputErrorDescription: true });
+    } else if (!validation.test(imgUrl)) {
+      this.setState({ inputErrorImgUrl: true });
+    } else if (!validation.test(imdbUrl)) {
+      this.setState({ inputErrorImdbUrl: true });
+    } else if (imdbId.length === 0) {
+      this.setState({ inputErrorImdbId: true });
+    } else {
+      this.props.addMovie({
+        title,
+        description,
+        imgUrl: imdbUrl,
+        imdbUrl,
+        imdbId,
+      });
+      this.setState({
+        title: '',
+        description: '',
+        imgUrl: '',
+        imdbUrl: '',
+        imdbId: '',
+        activeButton: true,
+      });
+    }
+  }
+
+  onTitleChange = (event) => {
+    if (this.state.title.length > 0) {
+      this.setState({ inputErrorTitle: false });
+    }
+
+    this.setState({
+      title: event.target.value,
+    });
+  }
+
+  onDescriptionChange = (event) => {
+    if (this.state.description.length > 0) {
+      this.setState({ inputErrorDescription: false });
+    }
+
+    this.setState({
+      description: event.target.value,
+    });
+  }
+
+  onImgUrlChange = (event) => {
+    if (this.state.imgUrl.length > 0) {
+      this.setState({ inputErrorImgUrl: false });
+    }
+
+    this.setState({
+      imgUrl: event.target.value,
+    });
+  }
+
+  onImdbUrlChange = (event) => {
+    if (this.state.imdbUrl.length > 0) {
+      this.setState({ inputErrorImdbUrl: false });
+    }
+
+    this.setState({
+      imdbUrl: event.target.value,
+    });
+  }
+
+  onImdbIdChange = (event) => {
+    if (this.state.imdbId.length > 0) {
+      this.setState({ inputErrorImdbId: false });
+    }
+
+    this.setState({
+      imdbId: event.target.value,
+    });
+  }
+
+  buttonSwitch = () => {
+    const { title, description, imdbId, imdbUrl, imgUrl } = this.state;
+
+    if (imdbId.length > 0
+        && title.length > 0
+        && description.length > 0
+        && imgUrl.length > 0
+        && imdbUrl.length > 0) {
+      this.setState({ activeButton: false });
+    }
+  }
+
+  blur = (event) => {
+    event.target.style.background = 'rgba(119, 156, 255, 0.1)';
+  }
 
   render() {
+    const { title, description, imgUrl, imdbUrl, imdbId } = this.state;
+
     return (
-      <form>
-        Put the form here
+      <form onSubmit={this.handleSubmit} onChange={this.buttonSwitch}>
+        <label htmlFor="inputTitle">Title:&nbsp;</label>
+        {this.state.inputErrorTitle
+        && (<span className="inputError">Please, enter the title!</span>)}
+        <input
+          id="inputTitle"
+          onBlur={this.blur}
+          className={`input${this.state.inputErrorTitle ? 'active' : ''}`}
+          value={title}
+          onChange={this.onTitleChange}
+        />
+        <label htmlFor="inputDescription">Description:&nbsp;</label>
+        {this.state.inputErrorDescription
+        && (<span className="inputError">Please, enter the description!</span>)}
```

Comment:
```suggestion
        && (
          <span className="inputError">Please, enter the description!</span>
        )}
```
it's very hard to read your code because of so much text in one place, try to add more space between logical blocks